### ISSUE TITLE
Drop Babel runtime

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/next-stream-page#1047]
As covered in dev huddle I think we should be looking to drop Babel at runtime. In the case of this app there is a strong reason to do so as it accounts for ~150mb memory usage which is a lot for a standard Heroku dyno.

- [ ] Upgrade to Node 6
- [ ] Setup pre-compilation of shared (JSX) files
- [ ] Refactor any features not-supported by latest Node into equivalent syntax (just module syntax?)
- [ ] Add precompilation step for JSX code
- [ ] Remove Babel runtime

**Comment by ironsidevsquincy** [comment]
`Add precompilation step for JSX code`

Wonder how much that will slow down development?

**Comment by i-like-robots** [comment]
Yep, something to consider... the lantern team precompile so worth checking out their setup.

**Comment by i-like-robots** [comment]
CC #1718
